js/src/components/contact-information/store-address-card.js

```diff
@@ -153,127 +125,57 @@ const StoreAddressCard = ( {
 			</div>
 		);
 	} else {
-		addressContent = <Spinner />;
+		addressContent = null;
 	}
 
-	if ( compactStyles ) {
-		description = (
-			<>
-				<p>
-					{ createInterpolateElement(
-						__(
-							'We’re using your store address from Woo Commerce settings for Google verification. This information won’t be public. Edit in <link>WooCommerce settings</link> if needed. Then, refresh to sync it to Google.',
-							'google-listings-and-ads'
-						),
-						{
-							link: settingsLink,
-						}
-					) }
-				</p>
-				{ addressContent }
-			</>
-		);
-	}
+	const longDescription = (
+		<p>
+			{ createInterpolateElement(
+				__(
+					'We’re using your store address for Google verification. This information won’t be public. Edit in <link>WooCommerce settings</link> if needed. Then, refresh to sync it to Google.',
+					'google-listings-and-ads'
+				),
+				{
+					link: settingsLink,
+				}
+			) }
+		</p>
+	);
+
+	const shortDescription = (
+		<p>
+			{ createInterpolateElement(
+				__(
+					'Your store address is required by Google for verification. This information won’t be public. Complete that in <link>WooCommerce settings</link>.',
+					'google-listings-and-ads'
+				),
+				{
+					link: settingsLink,
+				}
+			) }
+		</p>
+	);
+
+	const detail = (
+		<>
+			{ addressContent }
+			{ ! isAddressFilled && (
+				<ValidationErrors messages={ mapStoreAddressErrors( data ) } />
+			) }
+		</>
+	);
 
 	return (
 		<AccountCard
-			className={ classNames( 'gla-store-address-card', {
-				'gla-store-address-card--is-compact': compactStyles,
-			} ) }
+			className="'gla-store-address-card"
 			appearance={ APPEARANCE.ADDRESS }
 			alignIcon="top"
 			alignIndicator="top"
-			description={ description }
-			indicator={ refreshButton }
-		>
-			{ ! compactStyles && (
-				<>
-					<CardDivider />
-					<Section.Card.Body>
-						<Subsection.Title>
-							{ __( 'Store address', 'google-listings-and-ads' ) }
-						</Subsection.Title>
-						{ addressContent }
-						{ showValidation && (
-							<ValidationErrors
-								messages={ mapStoreAddressErrors( data ) }
-							/>
-						) }
-					</Section.Card.Body>
-				</>
-			) }
-		</AccountCard>
+			description={ isAddressFilled ? longDescription : shortDescription }
+			detail={ detail }
+			indicator={ showIndicator && refreshButton }
```

Comment:
@joemcgill On the settings panel, if the address has been filled and synced, the refresh button will not be rendered:
![image](https://github.com/user-attachments/assets/9c264d8c-6beb-427a-a141-e3b1c556ef84)

Reply:
Yes. That is intentional. There is no need to sync if the address is already synced.